File: aula20_08/app/model/ProcessoService.java
package aula20_08.app.model

import java.util.List;
import java.util.Random;

public class ProcessoService {

    public void ordenarBolha(List<Processo> lista) {
        if (lista == null || lista.size() <= 1) {
            return;
        }

        Processo tmp;
        boolean houveTroca;
        do {
            houveTroca = false;
            for (int i = 0; i < lista.size() - 1; i++) {
                
                if (lista.get(i).compareTo(lista.get(i + 1)) > 0) {
                    houveTroca = true;
                    tmp = lista.get(i);
                    lista.set(i, lista.get(i + 1));
                    lista.set(i + 1, tmp);
                }
            }
        } while (houveTroca);
    }

    public void popular(List<Processo> lista, int quantidade) {
        if (lista == null) return;

        Random gerador = new Random();
        for (int i = 0; i < quantidade; i++) {
            
            int idAleatorio = 100 + gerador.nextInt(400);
            lista.add(new Processo(idAleatorio, "gerando uma string " + gerador.nextInt(quantidade)));
        }
    }
}

File: python/app/model/processo_service.py
import random
from model.processo import processo

class processo_service:
    #classe para criar id aleatorio
    def popular(self, lista: list, quantidade: int):
        #se a lista n existe ele return a lista
        if lista is None:
            return
        for _ in range(quantidade):
            contador: int = quantidade
            id_aleatorio = random.randint(100, 499)
            desc = f"Gerando uma string {random.randint(0, quantidade -1)}"
            lista.append(processo(id_aleatorio, desc))

    #class para ordenar os IDs recebidos e deixar em ordem
    def ordenar_bolha(self, lista: list):
        if not lista or len(lista) <= 1:
            return

        n = len(lista)
        for i in range(n):
            houve_troca = False
            for j in range(0, n - i - 1):
                if lista[j+1] < lista[j]: #chama o __lt__
                    lista[j], lista [j+1] = lista[j+1], lista[j]
                    houve_troca = True
            if not houve_troca:
                break